src/pages/index.js

```diff
@@ -12,6 +12,8 @@ const IndexPage = () => {
     // console.log({ osat })
     setIp("")
     if (osat.loppu === "" || osat.loppu.length !== 4) return
+    if (osat.loppu.slice(0, 1) !== "2" && osat.loppu.slice(0, 1) !== "3") return
```

Comment:
It will allow only using 2 or 3 as first character

Reply:
This is needed because otherwise it will show any other than 2 as 7 when using VI vlan L2. Also it shouldnt calculate IP when ID is not valid.